iroha/test_network/tests/simeragi_with_mock.rs

```diff
@@ -47,7 +47,7 @@ pub mod utils {
 
```

Comment:
Wow. May be i messed up smth, but file is called simeragi... could you rename it?